### PR TITLE
meta: seeking stage 4 requires a link to the spec PR

### LINKED ISSUE #311: new requirement: proposals looking for stage 4 must include PR link?
We've had a few cases of champions bringing stage 3 proposals, looking for stage 4, without the PR reviewed by the editor, as required by our process. Requiring the PR link might help champions to not forget this step.

### PR DESCRIPTION
**Pull request by ljharb**
Fixes #311.

This is intended to serve mostly as a reminder, for easier cross-referencing.

### CODE REVIEW

File: 2019/01.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Deadline for advancement eligibility: [**January 19, 2019, 18:00 UTC**](https://
     1. Such proposals *should* include supporting materials when possible.
 1. Proposals looking to advance to stages 2, 3, or 4 must be added (and noted as such) *along with the necessary materials* prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
     1. Such proposals *must* link to supporting materials prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline. If these materials change substantially after the deadline, the proposal may be disqualified from advancement, based on the committee’s judgment.
+    1. Proposals looking to advance to stage 4 *must* link to a pull request into [the spec](https://github.com/tc39/ecma262), since the [process document](https://tc39.github.io/process-document/) requires one.
```
Comment (michaelficarra) [inline 2019/01.md]: Nit: it's the *process* that requires the PR, the document just wrote down the process. I would remove the word "document" here.

**Comment by littledan** [comment]
I would be OK with this being "should" (as long as the PR exists), but I don't have a problem with this PR.

**Comment by mathiasbynens** [comment]
+1 I’d prefer this to remain a “should”, but don’t feel too strongly.

**Comment by ljharb** [comment]
I'm going to merge this; more than happy to tweak or revert it if as desired.

**Pull request merged**